spec/lib/tasks/email_checkin_reminder_rake_spec.rb

```diff
@@ -9,12 +9,16 @@
   before(:each) do
     @upcoming = FactoryGirl.create(:checked_out_reservation,
                                    due_date: Time.zone.today)
+    @not_checked_out = FactoryGirl.build(:valid_reservation,
+                                         start_date: Time.zone.today - 1.day,
+                                         due_date: Time.zone.today)
+    @not_checked_out.save(validate: false)
     @not_upcoming = FactoryGirl.create(:valid_reservation,
                                        start_date: Time.zone.today + 1,
                                        due_date: Time.zone.today + 2)
   end
 
-  it 'sends emails for reservations that end today' do
+  it 'sends emails for checked-out reservations that end today' do
```

Comment:
Can we also write the inverse test to check that it *doesn't* send an e-mail for non-checked-out e-mails?